lua/kulala/ui/init.lua

```diff
@@ -278,6 +278,24 @@ M.show_headers_body = function()
   end
 end
 
+M.show_console = function()
```

Comment:
This is all from a different PR I guess.

Reply:
Sorry, I revised the commit. The pr contains only code formatting.